guides/BUILD.bazel

```diff
@@ -1,6 +1,6 @@
 package(default_visibility=["//visibility:public"])
 
-load("//tools/markdown-to-html:index.bzl", "markdown_to_html")
```

Comment:
I think it's more clear to import the file directly instead of adding the layer of indirection.

Reply:
I personally think it's still clear that way. The _big_ benefit is just that we don't need to change the import in hundreds of `BUILD.bazel` files. It's a pain keeping them in sync.

I'm okay switching to explicit paths, but personally don't think a layer of indirection is problematic here, since it helps reducing the boilerplate and makes it easier to sync the rule definitions across all build files.